integration-cli/docker_cli_help_test.go

```diff
@@ -79,19 +41,63 @@ func TestCmdHelpWidth(t *testing.T) {
 		lines := strings.Split(out, "\n")
 		for _, line := range lines {
 			if len(line) > 80 {
-				t.Fatalf("Help for %q is too long(%d chars):\n%s", command, len(line), line)
+				t.Fatalf("Line is too long(%d chars):\n%s", len(line), line)
 			}
-			if home != "" && strings.Contains(line, home) {
-				t.Fatalf("Help for %q should use ~ instead of %q on:\n%s", command, home, line)
+			if scanForHome && strings.Contains(line, `"`+home) {
+				t.Fatalf("Line should use ~ instead of %q:\n%s", home, line)
 			}
 		}
-	}
 
-	expected := 39
-	if count != expected {
-		t.Fatalf("Wrong # of commands (%d), it should be: %d\nThe list:\n%s",
-			len(cmds), expected, cmds)
+		// Make sure each cmd's help text fits within 80 chars and that
+		// on non-windows system we use ~ when possible (to shorten things).
+		// Pull the list of commands from the "Commands:" section of docker help
+		helpCmd = exec.Command(dockerBinary, "help")
```

Comment:
what are all the lines below this doing? isn't it duplicating the work done above when we looped through the lines?